src/app/view/components/login/dot-login-component/login-container.component.ts

```diff
@@ -35,34 +31,36 @@ export class LoginContainerComponent {
 
     constructor(
         private dotRouterService: DotRouterService,
-        private httprequestUtils: HttpRequestUtils,
         private loginService: LoginService,
         private dotLoadingIndicatorService: DotLoadingIndicatorService,
-        private loggerService: LoggerService
-    ) {
-        // this.dotLoadingIndicatorService.hide();
-        // TODO: change the httpRequestUtils.getQueryParams() with an NG2 method equivalent to QueryParams on NGRX.
-        const queryParams: Map<string, any> = this.httprequestUtils.getQueryParams();
-        if (<boolean>queryParams.get('changedPassword')) {
-            this.passwordChanged = queryParams.get('changedPassword');
-        } else if (<boolean>queryParams.get('resetEmailSent')) {
-            this.resetEmailSent = queryParams.get('resetEmailSent');
-            this.resetEmail = decodeURIComponent(queryParams.get('resetEmail'));
-        }
+        private loggerService: LoggerService,
+        private activatedRoute: ActivatedRoute
+    ) {}
+
+    ngOnInit() {
+        this.activatedRoute.queryParams.pipe(take(1)).subscribe((params: Params) => {
+            if (params['changedPassword']) {
+                this.passwordChanged = params['changedPassword'];
+            } else if (params['resetEmailSent']) {
+                this.resetEmailSent = params['resetEmailSent'];
+                this.resetEmail = decodeURIComponent(params['resetEmail']);
+            }
+        });
```

Comment:
maybe we can handle this with just one parameter something like:

comeFrom='changedPassword'
comeFrom='resetEmailSent'